tests/persist.rs

```diff
@@ -4,48 +4,103 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-#[cfg(feature = "persistance")]
-mod persist {
+use nstack::NStack;
 
-    use microkelvin::{
-        BackendCtor, Compound, DiskBackend, PersistError, Persistance,
-    };
-    use nstack::NStack;
+use std::io;
 
-    #[test]
-    fn persist_across_threads() -> Result<(), PersistError> {
-        let n: u64 = 1024;
+use rend::LittleEndian;
 
-        let mut stack = NStack::<u64, ()>::new();
+use microkelvin::{ArchivedCompound, Cardinality, Nth, Portal};
 
-        for i in 0..n {
-            stack.push(i)?;
-        }
+fn persist() -> Result<(), io::Error> {
+    let n: u64 = 1024;
+
+    let mut stack = NStack::<_, Cardinality>::new();
+    for i in 0..n {
+        let i = LittleEndian::from(i);
```

Comment:
The endianness is actually not neccesary here.

Remove this line, and remove the `*` - deref below and it works with less complexity.

Reply:
Removed LittleEndian use in all tests